assets/js/pages/ChecksCatalog/ChecksCatalog.jsx

```diff
@@ -26,19 +34,27 @@ const providerOptionRenderer = createOptionRenderer(
 
 const clusterTypeRenderer = createOptionRenderer(
   'All cluster types',
-  (clusterType, disabled) => (
-    <>
-      {getClusterTypeLabel(clusterType)}
-      {disabled && (
-        <Pill
-          size="xs"
-          className="absolute right-2 bg-green-100 text-green-800"
-        >
-          Coming Soon
-        </Pill>
-      )}
-    </>
-  )
+  ({ type, hanaScenario }, disabled) => {
+    const clusterTypeLabel = [
```

Comment:
I think we can make this more obvious if we just want to remove the trailing empty space.
Simply: 
```
// trim comes from lodash
{trim(`${getClusterTypeLabel(type)} ${getClusterScenarioLabel(hanaScenario)}`)}
```